app/models/ems_azure.rb

```diff
@@ -0,0 +1,49 @@
+
+class EmsAzure < EmsCloud
```

Comment:
We should follow the new provider plugin pattern. This class will be `class ManageIQ::Providers::Microsoft::CloudManager < ManageIQ::Providers::CloudManager`